content/english/events/2019-12-12-caminos-into-tech.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Caminos into Tech"
+description: "A conversation with Latinx tech professionals"
+date: 2019-12-12
+image: "/assets/img/events/2019-12-12-honey-panel.jpg"
```

Comment:
image files should be less than 1MB in size - can you use an online image compressor to reduce the image size - making the width max 1500px helps a lot as well!

right now, the image is almost 5MB which is very big and is not good for the performance of website